test/gtest/uct/ib/test_sockaddr.cc

```diff
@@ -112,6 +112,53 @@ UCS_TEST_P(test_uct_sockaddr, connect_client_to_server)
      * test ends and the client's ep was destroyed */
 }
 
+UCS_TEST_P(test_uct_sockaddr, many_clients_to_one_server)
+{
+    UCS_TEST_MESSAGE << "Testing " << ucs::sockaddr_to_str(sock_addr.addr);
+    server_recv_req = 0;
+    err_count = 0;
+
+    uct_iface_params client_params;
+    entity *client_test;
+    int i, num_clients = 100;
+
+    /* multiple clients, each on an iface of its own, connecting to the same server */
+    for (i = 0; i < num_clients; ++i) {
+        /* open iface for the client side */
+        memset(&client_params, 0, sizeof(client_params));
+        client_params.open_mode       = UCT_IFACE_OPEN_MODE_SOCKADDR_CLIENT;
+        client_params.err_handler     = err_handler;
+        client_params.err_handler_arg = reinterpret_cast<void*>(this);
+
+        client_test = uct_test::create_entity(client_params);
+        m_entities.push_back(client_test);
+
+        client_test->connect(i, *server, 0);
+    }
+
+    while (server_recv_req < num_clients);
```

Comment:
i would sched_yield inside the loop to let progress thread run

Reply:
won't it run anyway? also, should a test be aware of the progress thread...?